cfme/fixtures/multi_region.py

```diff
@@ -0,0 +1,75 @@
+import attr
+import pytest
+
+
+@attr.s
+class ApplianceCluster(object):
```

Comment:
Can you add a small docblock for this class?

Reply:
sure, on it